SysBackup/README.md

```diff
@@ -0,0 +1,65 @@
+SysBackup
+================
+
+This is a system for backup and restore for two components of Zenko for 
+disaster recovery purposes:
+* Metadata in MongoDB
+* Zookeeper queues
+
+Currently the backup process is automated. The process/script for backup will 
+be run as a cronjob inside the container: zenko-backup.
+
+MongoDB metadata backup
+-----------------------
+
+The tool used to dump mongoDB documents and collections is `mongodump`. The
+backup will be a combination of a fullbackup and multiple incremental backups.
+Everything will be automated inside the container zenko-backup using a cronjob.
+The cronjob will take a full backup everyday using the below command:
+
+```shell
+mongodump --host $MONGODB_HOST --port $MONGODB_PORT --oplog --gzip -o \
+    $BACKUP_DATA_PATH/mongodb-backup/`date \+\%Y\%m\%d_\%s`
+```
+
+This command will assume that MONGODB_HOST, MONGODB_PORT and BACKUP_DATA_PATH
+environment variables are defined in the help chart.
+
+For incremental backup the script `incrementalMongodbBackup.sh` will be used.
+This is a simple bash script to analyze the full backups or previous
+incremental backups and create backups post the previous entry. It will avoid
```

Comment:
I'm having trouble understanding "create backups post the previous entry" - did you mean something like "create backups starting from the most recent backup entry"?